pyproject.toml

```
@@ -33,22 +33,19 @@ classifiers = [
     "Intended Audience :: Science/Research",
     "License :: OSI Approved :: MIT License",
     "Operating System :: OS Independent",
-    "Programming Language :: Python",
-    "Programming Language :: Python :: 3.8",
-    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3",
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
     "Topic :: Scientific/Engineering",
 ]
 
 dependencies = [
-    "numpy >=1.15",
-	"scipy>=1.0",
-    "vtk>=9.1.0",
-    "pillow>=5.4.1",
-    "packaging >=17.0",
-    "pygltflib>=1.15.3",
-    "aiohttp>=3.8.4",
+    "numpy>=2.0",
+    "scipy>=1.13.0",
+    "pillow>=10.4.0",
+    "packaging >=23.0",
```

remove spacing from the 
```suggestion
    "packaging>=23.0",
```